### DISCUSSION TITLE
[M-Z] (17 Giugno 2019) Esercizio 3

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#445] [Esami & Esoneri M-Z]
### Traccia

<img width="612" alt="image" src="https://github.com/user-attachments/assets/daacd4bc-8a6d-4204-aba0-af54e5dc9a7c">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20190617.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20190617.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/9018106d-bbfe-42c6-b052-cb6ddbf8063c)
